### ISSUE TITLE
Support for IPv6-only networks

### ISSUE THREAD

**Issue by bcosma** [fkuehne/upnpx#82]
Apple announced that starting with 1st of June 2016 all apps must support IPv6 only networks. 
Is there any chance to have this in upnpx too?

**Comment by fkuehne** [comment]
upnpx works just fine with IPv6 and our last VLC-iOS update was just approved. Note quite sure what you need.

**Comment by bcosma** [comment]
I am talking about this [issue](https://developer.apple.com/library/mac/documentation/NetworkingInternetWeb/Conceptual/NetworkingOverview/UnderstandingandPreparingfortheIPv6Transition/UnderstandingandPreparingfortheIPv6Transition.html#//apple_ref/doc/uid/TP40010220-CH213-SW1).

We tested the UPNPX library in a ipv6-only network and it didn't return any result.

**Comment by fkuehne** [comment]
Indeed. You are absolutely correct. I don't have capacities to address this at this point, sorry.
